### PR TITLE
pagination validation

### PR DESCRIPTION
**Pull request by evgeniilezhnin**

### CODE REVIEW

File: backend/src/main/java/ai/verta/modeldb/versioning/VersioningServiceImpl.java [hunk 1]
```diff
@@ -54,6 +54,14 @@ public void listRepositories(
     try {
       try (RequestLatencyResource latencyResource =
           new RequestLatencyResource(modelDBAuthInterceptor.getMethodName())) {
+        if (request.hasPagination()) {
+          if (request.getPagination().getPageNumber() < 1) {
+            throw new ModelDBException("Page number is invalid", Code.INVALID_ARGUMENT);
+          }
+          if (request.getPagination().getPageLimit() < 1) {
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/versioning/VersioningServiceImpl.java]: Also limit the max to 100.

File: backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java [hunk 2]
```diff
@@ -254,8 +254,10 @@ private RepositoryEntity getRepositoryById(Session session, RepositoryIdentifica
               workspaceDTO.getWorkspaceType(),
               false);
       int pageLimit = request.getPagination().getPageLimit();
```
Comment (conradoverta) [inline backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java]: What happens here if no pagination is provided? Does it throw an error?

Comment (ravishetye) [reply backend/src/main/java/ai/verta/modeldb/versioning/RepositoryDAORdbImpl.java]: there will be a default object with empty stuff, so not error

File: backend/src/main/java/ai/verta/modeldb/versioning/VersioningServiceImpl.java (hunk 1 above)
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/versioning/VersioningServiceImpl.java]:
```suggestion
          if (request.getPagination().getPageLimit() < 1 &&
              request.getPagination().getPageLimit() > 100) {
```

**Pull request merged**